apollo-router/src/configuration/migrations/0038-ignored-headers-subs-dedup.yaml

```diff
@@ -0,0 +1,5 @@
+description: "`subscription.enable_deduplication` was a true/false before but now has child properties `enabled` and `ignored_headers`"
```

Comment:
Can we still do this? I thought we now prevent starting the router with non-migrated config, so this would be a breaking change https://github.com/apollographql/router/pull/6777

Reply:
Waiting for https://github.com/apollographql/router/pull/7162 to be merged before